app/components/views/GetStartedPage/CreateWallet/CreateWalletForm/ConfirmSeed/Form.js

```diff
@@ -16,17 +16,23 @@ export const ConfirmSeedForm = ({ seedWords, onChangeSeedWord, onReturnToNewSeed
       </div>
       <div className="seedArea">
         {seedWords.map((seedWord) => {
-          const className = "seedWord " + (!seedWord.show ? seedWord.match ? "match" : "no-match" : "");
-          return ( seedWord.show ?
-            <div key={seedWord.index} className="seedWord">{seedWord.word}</div> :
-            <SingleSeedWordEntry
-              className={className}
-              disabled={seedWord.show}
-              onChange={onChangeSeedWord}
-              seedWord={seedWord}
-              value={{ name: seedWord.word }}
-              key={seedWord.index}
-            />);
+          const className = "seedWord " + ((!seedWord.show && seedWord.word != "") ? seedWord.match ? "match" : "no-match" : seedWord.show ? "filled" : "empty" );
```

Comment:
I think this is hard to understand. If you check if the word needs to be shown first, would be easier to understand, I think. For example: 

`seedWord.show ? "filled" : seedWord.match ? "match" : "no-match" : "empty"`